docs/guides/executor.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Executor primitive (beta)
+description: Executor is a lower-level primitive that provides more flexibility when customizing your error mitigation workflow.
+
+---
+
+# Executor primitive (beta)
+
+<Admonition type="note" title="Beta release">
+The Executor primitive is currently in beta and might not be stable. You are invited to test it and provide feedback. (((how?)))
```

Comment:
It's an open beta, so anyone can access it. They just need to install the preview branch.

Reply:
ah sorry I misunderstood this. I thought it's asking how do they get invited 😄. 

The best way to provide feedback is probably to open issues in samplomatic or qiskit-ibm-runtime repo.